src/rgw/rgw_tools.h

```diff
@@ -268,4 +268,5 @@ using RGWDataAccessRef = std::shared_ptr<RGWDataAccess>;
 /// calls and error handling.
 void rgw_complete_aio_completion(librados::AioCompletion* c, int r);
 
+std::map<std::string, ceph::buffer::list>* no_change_attrs();
```

Comment:
worth a comment that this returns a static non-null address, recognized only by rgw_put_system_obj() and not the system obj services

Reply:
Done!